website/docs/cdktf/release/upgrade-guide-v0-9.mdx

```diff
@@ -0,0 +1,38 @@
+---
+page_title: Upgrading to CDKTF Version 0.9
+description: >-
+  We changed some provider bindings to work better across all languages.
+---
+
+# Upgrading to CDK for Terraform Version 0.9
+
+We changed some provider bindings to work better across all languages.
+
+### Standardize IResolvable Usage [#1299](https://github.com/hashicorp/terraform-cdk/pull/1299)
+
+This is an effort to make sure attributes can be freely passed between resources for all different types.
+
+There is a minor breaking change:
+
+- `count` on resources and data sources has gone from `number | cdktf.IResolvable` to `number`. If code was previously passing an `IResolvable`, it will now need to use `Token.asNumber()` on Typescript and Java , `Token.AsNumer()` on C#, `cdktf.Token_AsNumber()` on Go, and `Token().as_string()` on Python.
```

Comment:
```suggestion
- `count` on resources and data sources has gone from `number | cdktf.IResolvable` to `number`. If code was previously passing an `IResolvable`, it will now need to wrap the `IResolvable` using `Token.asNumber()` on Typescript and Java , `Token.AsNumer()` on C#, `cdktf.Token_AsNumber()` on Go, and `Token().as_string()` on Python.
```